pootle/core/search/backends/elasticsearch.py

```diff
@@ -75,6 +92,10 @@ def search(self, unit):
             }
         )
 
+        # Sometimes urllib seems to return empty string so handle this here
```

Comment:
@julen - agreed im going to add a log.exception here

with urllib debug logging you can see a connection timeout but somewhere along the line it gets swallowed and returns status 200 + empty body